### DISCUSSION TITLE
Limit parsing to limit value?

### DISCUSSION THREAD

**Discussion by edrock200** [meisnate12/Plex-Meta-Manager#875] [Q&A]
Bit a random question/suggestion. I have an IMDB list with 3500 or so entries, and I only want the first 500. I have the URL configured to sort by release date so the first 500 are the newest movies, which is how I want it. I noticed when PMM processes the list, it parses all 3500 entries before syncing the 500 entries. Is there a way to tell PMM not to parse beyond the limit value, or is there a benefit to allowing it to parse all the entries even if a limit is set?

**Comment by YozoraXCII** [comment]
At a guess, it needs to parse the full thing because if you set a limit of 500, you could end up with half of that number if you don’t have the media. So it parses the full thing and then adds until the limit is reached or the list ends (whichever comes first)

**Comment by meisnate12** [comment]
can you post the collection definition

**Reply by edrock200** [reply]
It is just a trakt list. @YozoraXCII 's reply makes sense. But in the event a list is say, 10k entries, and I only want my collection to be 500 entries, would be nice to have a parse limit. e.g. I could set it to 1000 or 1500.

**Reply by meisnate12** [reply]
thats why i asked for the definition because theres more then one limit you can limit the whole collection or limit the individual builder (for some builders) if you limit the builder it will only parse whats needed
